src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+html {
+  color: #3d4e61;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+body {
+  margin-block: 0;
```

Comment:
Using 'margin-block' is not a common practice and may not be supported in all browsers. It's better to stick with 'margin-top' and 'margin-bottom' for clarity and broader compatibility.